template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/databricks.yml.tmpl

```diff
@@ -13,26 +13,16 @@ variables:
 
 
 include:
-  # Include ML artifact resources for the ml project
-  # Defines model and experiment
-  - ./databricks-resources/ml-artifacts-resource.yml
-
-  # Include workflow resources of the ml project
-  # Defines workflow for model training -> validation -> deployment
-  - ./databricks-resources/model-workflow-resource.yml
-  {{- if (eq .input_include_feature_store `yes`) }}
-  # Defines workflow for feature engineering
-  - ./databricks-resources/feature-engineering-workflow-resource.yml{{ end }}
-  # Defines workflow for scheduled batch inference
-  - ./databricks-resources/batch-inference-workflow-resource.yml
-  # Defines workflow for data monitoring, metric refresh, alerts and triggering retraining
-  - ./databricks-resources/monitoring-workflow-resource.yml
-
+  # Resources folder contains ML artifact resources for the ml project that defines model and experiment
+  # And workflows resources for the ml project including model training -> validation -> deployment, 
+  # feature engineerng, batch inference, data monitoring, metric refresh, alerts and triggering retraining
+  - ./resources/*
```

Comment:
There are test failures. Looks like it also tried to import the README.md as yaml file and failed.
Could you try `./resources/*.yml`?
If it doesn't work, we can revert back to specifying each file.